robocop/checkers/misc.py

```diff
@@ -601,6 +607,78 @@
 
         """,
     ),
+    "0927": Rule(
+        RuleParam(
+            name="sections_order",
+            default="documentation,tags,timeout,setup,template,keyword,teardown",
+            converter=parse_test_case_order_param,
+            show_type="str",
```

Comment:
You have used most of our misc stuff for creating rules - good to see that others can do it :) (meaning it's not dead code)